modules/Sources/Features/NHSendFlow/NHSendFlowStore.swift

```diff
@@ -172,6 +194,48 @@ public struct NHSendFlowReducer: ReducerProtocol {
                 let _ = state.path.popLast()
                 return .none
             case .path(.element(id: _, action: .review(.sendZcashTapped))):
+                if nhUserPreferencesStorage.areBiometricsEnabled() {
```

Comment:
I'm leaving updating the test suite as a TODO for now, since I am planning on cleaning up a lot of code, moving things around, and upgrading to TCA 1.0. So it doesn't really make sense to put the cycles into the tests when those changes are yet to come that will end up breaking whatever tests I write.

Reply:
Fair enough!

Although for this specific case I'd either make the effort of having some test that will break and act as a reminder of the thing you need to do, or mark it with a specific TODO item linked to a github issue that's blocking to your release process.

I insist mostly because if there's a bug. triggered on certain Authentication setups, it would lock users' funds unless they change their system's biometric settings.